Update eip-725.md (#3865)
ethereum/EIPs: EIPS/eip-725.md

> This standard requires [ERC173](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-173.md) and should implement the functions:

Reviewer: ```suggestion
This standard requires [ERC-173](./eip-173.md) and should implement the functions:
```

- `owner() view`

> - `transferOwnership(address newOwner)` <br>

Reviewer: ```suggestion
- `transferOwnership(address newOwner)`
```

    event ContractCreated(address indexed contractAddress);
    event Executed(uint256 indexed operation, address indexed to, uint256 indexed  value, bytes data);

    function execute(uint256 operationType, address to, uint256 value, bytes memory data) external;
}

//ERC165 identifier: `0x5a988c0f`
interface IERC725Y /* is ERC165, ERC173 */ {
    event DataChanged(bytes32 indexed key, bytes value);

    function getData(bytes32[] calldata _keys) external view returns(bytes[] memory);

>     function setData(bytes32[] calldata _keys, bytes[] calldata _values) external ;

Reviewer: ```suggestion
    function setData(bytes32[] calldata _keys, bytes[] calldata _values) external;
```
